tests/test_integration_crashpad.py

```diff
@@ -1,15 +1,22 @@
 import pytest
 import os
+import sys
 import time
 from . import make_dsn, run, Envelope
 from .conditions import has_crashpad
 from .assertions import assert_session
 
+if not has_crashpad:
+    pytest.skip("test needs crashpad backend", allow_module_level=True)
```

Comment:
Did you consider the module-level skipif mark instead as the docs suggest?

https://docs.pytest.org/en/stable/example/markers.html#marking-whole-classes-or-modules

Was that not suitable for some reason?